datafusion/expr/src/logical_plan/builder.rs

```diff
@@ -279,6 +281,14 @@ impl LogicalPlanBuilder {
         )?)))
     }
 
+    pub fn prepare(&self, name: String, data_types: Vec<DataType>) -> Result<Self> {
```

Comment:
Perhaps we can add a docstring to this method

Reply:
Done